hpx/parallel/algorithms/move.hpp

```diff
@@ -119,41 +133,18 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v1)
     ///           element in the destination range, one past the last element
     ///           copied.
     ///
```

Comment:
Please adapt the documentation to reflect the change in return type for the `parallel::move()` algorithm.